### PR TITLE
[CARBONDATA-426] replace if else with conditional operator

### PR DESCRIPTION
**Pull request by PallaviSingh1992**
Replaced the if-else with Conditional operator. 
Reduced the number of variables used.
Reduced the boiler plating by removing redundant conditional checks.

### CODE REVIEW

**Comment by sujith71955** [comment]
Please rebase the code

File: integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java [hunk 1]
```diff
@@ -198,21 +183,16 @@ public static CompactionType determineCompactionType(String metaFolderPath) {
...
-    if (compactionType.equals(CompactionType.MINOR_COMPACTION)) {
-      compactionRequiredFile = metaFolderPath + CarbonCommonConstants.FILE_SEPARATOR
-          + CarbonCommonConstants.minorCompactionRequiredFile;
-    } else {
-      compactionRequiredFile = metaFolderPath + CarbonCommonConstants.FILE_SEPARATOR
-          + CarbonCommonConstants.majorCompactionRequiredFile;
-    }
+                                                     CompactionType compactionType) {
+    String file = metaFolderPath + CarbonCommonConstants.FILE_SEPARATOR;
+    String compactionRequiredFile = compactionType.equals(CompactionType.MINOR_COMPACTION) ?
```
Comment (sujith71955) [inline integration/spark/src/main/java/org/apache/carbondata/integration/spark/merger/CarbonCompactionUtil.java]: Enum comparison better to use ==

File: integration/spark-common/src/main/java/org/apache/carbondata/spark/merger/CarbonCompactionUtil.java [hunk 2]
```diff
@@ -142,20 +142,14 @@ private static void groupCorrespodingInfoBasedOnTask(TableBlockInfo info,
...
-        + CarbonCommonConstants.majorCompactionRequiredFile;
+    String statusFile = metaFolderPath + CarbonCommonConstants.FILE_SEPARATOR;
     try {
-      if (FileFactory.isFileExist(minorCompactionStatusFile,
-          FileFactory.getFileType(minorCompactionStatusFile)) || FileFactory
-          .isFileExist(majorCompactionStatusFile,
-              FileFactory.getFileType(majorCompactionStatusFile))) {
-        return true;
-      }
+      return (FileFactory.isFileExist(statusFile + CarbonCommonConstants.minorCompactionRequiredFile,
```
Comment (PallaviSingh1992) [inline integration/spark-common/src/main/java/org/apache/carbondata/spark/merger/CarbonCompactionUtil.java]: I will revert back to original code

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

File: integration/spark-common/src/main/java/org/apache/carbondata/spark/merger/CarbonCompactionUtil.java [hunk 3]
```diff
@@ -142,20 +142,15 @@ private static void groupCorrespodingInfoBasedOnTask(TableBlockInfo info,
...
-    String majorCompactionStatusFile = metaFolderPath + CarbonCommonConstants.FILE_SEPARATOR
-        + CarbonCommonConstants.majorCompactionRequiredFile;
+    String statusFile = metaFolderPath + CarbonCommonConstants.FILE_SEPARATOR;
     try {
-      if (FileFactory.isFileExist(minorCompactionStatusFile,
-          FileFactory.getFileType(minorCompactionStatusFile)) || FileFactory
-          .isFileExist(majorCompactionStatusFile,
-              FileFactory.getFileType(majorCompactionStatusFile))) {
+      if (FileFactory.isFileExist(statusFile + CarbonCommonConstants.minorCompactionRequiredFile,
+          FileFactory.getFileType(statusFile + CarbonCommonConstants.minorCompactionRequiredFile))
```
Comment (jackylk) [inline integration/spark-common/src/main/java/org/apache/carbondata/spark/merger/CarbonCompactionUtil.java]: indentation is wrong, should add two space here

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/154/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/156/

File: integration/spark-common/src/main/java/org/apache/carbondata/spark/merger/CarbonCompactionUtil.java (hunk 3 above)
Comment (PallaviSingh1992) [inline integration/spark-common/src/main/java/org/apache/carbondata/spark/merger/CarbonCompactionUtil.java]: Used java-code-format-template.xml provided in the dev folder to format the code

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/157/

**Pull request closed without merging**